platform/barefoot/sonic-platform-modules-imt/debian/control

```diff
@@ -0,0 +1,11 @@
+Source: sonic-platform-modules-imt
+Section: main
+Priority: extra
```

Comment:
As far as I know package priority 'extra' is deprecated.

Reply:
fixed